data/profiles/demo-auth.yaml

```diff
@@ -0,0 +1,413 @@
+          readinessFailureThreshold: 30
+          includeIPRanges: "*"
+          autoInject: enabled
+          tracer: "zipkin"
+
+# Policy feature
+policy:
+  enabled: true
+  components:
+    namespace: istio-policy
+    policy:
+      common:
+        k8s:
+          replicaCount: 1
+          resources:
+            requests:
+              cpu: 10m
+              memory: 100Mi
+          env:
+            - name: GODEBUG
+              value: "gctrace=1"
+        values:
+          image: mixer
+          autoscaleEnabled: false
+          adapters:
+            kubernetesenv:
+              enabled: true
+
+# Telemetry feature
+telemetry:
+  enabled: true
+  components:
+    namespace: istio-telemetry
+    telemetry:
+      common:
+        k8s:
+          env:
+          - name: GODEBUG
+            value: "gctrace=1"
+          - name: GOMAXPROCS
+            value: "6"
+          replicaCount: 1
+          resources:
+            requests:
+              cpu: 50m
+              memory: 100Mi
+        values:
+          image: mixer
+          autoscaleEnabled: false
+          sessionAffinityEnabled: false
+          loadshedding:
+            mode: enforce
+            latencyThreshold: 100ms
+          adapters:
+            stdio:
+              enabled: true
+              outputAsJson: false
+            prometheus:
+              enabled: true
+              metricsExpiryDuration: 10m
+            kubernetesenv:
+              enabled: true
+            stackdriver:
+              enabled: false
+            useAdapterCRDs: false
+
+# Security feature
+security:
+  enabled: true
+  controlPlaneMtls: true
+  dataPlaneMtlsStrict: true
+  components:
+    namespace: istio-security
+
+    citadel:
+      common:
+        values:
+          image: citadel
+          selfSigned: true # indicate if self-signed CA is used.
+          trustDomain: cluster.local # indicate the domain used in SPIFFE identity URL
+          dnsCerts:
+            istio-sidecar-injector-service-account.istio-remote: istio-sidecar-injector.istio-remote.svc
+            istio-sidecar-injector-service-account.istio-pilot11: istio-sidecar-injector.istio-pilot11.svc
+            istio-sidecar-injector-service-account.istio-control: istio-sidecar-injector.istio-control.svc
+            istio-sidecar-injector-service-account.istio-master: istio-sidecar-injector.istio-master.svc
+            istio-sidecar-injector-service-account.istio-control-master: istio-sidecar-injector.istio-control-master.svc
+            istio-galley-service-account.istio-pilot11: istio-galley.istio-pilot11.svc
+            istio-galley-service-account.istio-control: istio-galley.istio-control.svc
+            istio-galley-service-account.istio-master: istio-galley.istio-master.svc
+            istio-galley-service-account.istio-control-master: istio-galley.istio-control-master.svc
+            istio-galley-service-account.istio-config: istio-galley.istio-config.svc
```

Comment:
These cert names correspond to the dual-control plane topology. I think Citadel includes defaults for the basic case already. We _might_ need `istio-pilot-service-account.istio-system:istio-pilot.istio-system` for parity with the istio/istio manifests.

Reply:
Done

Reply:
This doesn't look right. For the default/demo installs in istio-system namespace I don't think any of these are needed. IIUC, these additional certs are required for the dual control plane install.

Reply:
Yep, removed other items and only added `istio-pilot-service-account.istio-system:istio-pilot.istio-system`.